data/json/recipes/weapon/ranged.json

```diff
@@ -734,6 +734,23 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "stick", 3 ], [ "2x4", 2 ] ], [ [ "bone_sturdy", 3, "LIST" ] ], [ [ "cordage_superior", 2, "LIST" ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "reflexbow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "archery", 2 ] ],
+    "difficulty": 4,
+    "time": "8 h",
```

Comment:
This is ludicrous, and the primary reason I didn't add this recipe myself, a composite bow takes many, many hours to construct, estimates place it at a week of work if you neglect drying/curing time for the adhesives.

Reply:
On this we fully agree. Again, as I mentioned above, the recipe mostly just repeats what's already in the game for the heavy composite bow and I feel like a lot of recipes in general are done at unreasonable speed. I appreciate the effort to move away from that. 

I'm happy to put in a little research and update both recipes to take longer. I can even set it up to include intermediary products which have to sit and cure, like the tanning hide or the juvenile sourdough starter.